checkpoints/checkpoint2.1/index.html

```diff
@@ -0,0 +1,71 @@
+<head> 
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
```

Comment:
The page has some slight issues when its not wide enough to show your top bar.
But over all nice job using bootstrap